gitpractice/task2/src/app/userModel.ts

```diff
@@ -0,0 +1,15 @@
+export class User{
```

Comment:
this is a model, you should maintain only an interface of users as said in the task